tests/Operation/ModifyCollectionFunctionalTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Operation\ModifyCollection;
+use MongoDB\Operation\CreateCollection;
+use MongoDB\Operation\CreateIndexes;
+
+class ModifyCollectionFunctionalTest extends FunctionalTestCase
+{
+    public function testCollMod()
+    {
+        $operation = new CreateCollection($this->getDatabaseName(), $this->getCollectionName());
+        $operation->execute($this->getPrimaryServer());
+
+        $indexes = [['key' => ['lastAccess' => 1], 'expireAfterSeconds' => 3]];
+        $createIndexes = new CreateIndexes($this->getDatabaseName(), $this->getCollectionName(), $indexes);
+        $createIndexes->execute($this->getPrimaryServer());
+
+        $keyPattern = (object) ['lastAccess' => 1];
```

Comment:
Likewise, not sure if `(object)` cast is needed. If not, can we just use the `['lastAccess' => 1]` literal below and do without this variable?